test/e2e/index-test.js

```diff
@@ -0,0 +1,18 @@
+import IndexPage from './page-model';

```

Comment:
Rename to "main.password.spec.js" to follow convention for other tests.

main.{fixture}.spec.js for all tests, to separate concerns.